inc/search.class.php

```diff
@@ -7838,13 +8029,12 @@ static function explodeWithID($pattern, $subject) {
     *
     * @return string
     */
-   public static function joinDropdownTranslations($alias, $table, $itemtype, $field) {
-      return "LEFT JOIN `glpi_dropdowntranslations` AS `$alias`
-                  ON (`$alias`.`itemtype` = '$itemtype'
-                        AND `$alias`.`items_id` = `$table`.`id`
-                        AND `$alias`.`language` = '".
-                              $_SESSION['glpilanguage']."'
-                        AND `$alias`.`field` = '$field')";
+   public function joinDropdownTranslations($alias, $table, $itemtype, $field) {
```

Comment:
1. Should be mentionned in changelog.

2. This method is only privately used, may be changed to private.